### PR TITLE
[QuicTransport] Address comments in pull/22844

### PR DESCRIPTION
**Pull request by yutakahirano**
Address comments provided in
https://github.com/web-platform-tests/wpt/pull/22844.

### CODE REVIEW

**Comment by yutakahirano** [comment]
@vasilvv @ricea can you take a look?

File: tools/quic/quic_transport_server.py [hunk 1]
```diff
@@ -56,9 +56,11 @@ def quic_event_received(self, event: QuicEvent) -> None:
                     isinstance(event, StreamDataReceived) and
                     event.stream_id == 2):
                 # client indication process
+                prefix = 'Client inditation error: '
```
Comment (ricea) [inline tools/quic/quic_transport_server.py:59]: s/inditation/indication/

**Review by ricea — APPROVED** [review]
lgtm

**Review by vasilvv — APPROVED** [review]
LGTM.

File: tools/quic/quic_transport_server.py (hunk 1 above)
Comment (yutakahirano) [reply tools/quic/quic_transport_server.py]: Done.

**Comment by yutakahirano** [comment]
Thanks!

**Pull request merged**